src/components/file-upload/file-upload.ts

```diff
@@ -60,11 +60,12 @@ export class MFileUpload extends ModulVue {
     public maxSizeKb?: number;
     @Prop()
     public maxFiles?: number;
-
     @Prop({
         default: DEFAULT_STORE_NAME
     })
     public storeName: string;
+    @Prop()
+    public open: boolean;
```

Comment:
Je n'ai pas essayé localement, mais est-ce que le téléverseur peut se fermer par lui-même (genre cliquer sur un X dans le dioalogue)? Si oui, est-ce que le prop Open se met à false? Sinon c'est un peu incohérent...

Reply:
Mon dernier commit corrige le problème!

Reply:
Ça va fonctionner à l'interne. Mais si je fais ceci:
```
// .ts
private o: boolean = true;

// .html
<m-file-upload :open.sync="o"><m-file-upload>
```
la valeur de `o` ne sera pas mise à jour à la fermeture. Il manque un `this.$emit('open.update', value)` lorsqu'on set propOpen sur le close ou cancel. La variable `propOpen` devrait s'appeler `internalOpen`. `propOpen` devrait avoir un getter (qui lit Open ou internalOpen si Open est undefined) et un setter (qui set internalOpen et fait le $emit update). Le watch ne fait qu'assigner la valeur internalOpen sans passer par le setter.